src/views/components/model.jsx

```diff
@@ -133,7 +135,7 @@ export default class Model extends React.Component {
                     </div>
                 </div>
                 <div className="actionArea">
-                    <img className='btnAddItemToModel' onClick={this.showAddItemConfirmModal.bind(this)} src="../assets/images/add.svg"/>
+                    <img id={`addbtn${this.state.model.address}`} className='btnAddItemToModel' onClick={this.showAddItemConfirmModal.bind(this)} src="../assets/images/add.svg"/>
```

Comment:
I mean, `#{someModelAddr} .actionArea img[src*="add"]` is pretty nice selector too&hellip;
You don't even need the `.actionArea` if you don't want it&hellip;